langchain/vectorstores/chroma.py

```diff
@@ -314,6 +320,72 @@ def delete_collection(self) -> None:
         """Delete the collection."""
         self._client.delete_collection(self._collection.name)
 
+    # should take document or list of documents
+    def update(self, documents: List[Document] | Document) -> None:
```

Comment:
nit: think `|` type hint operator only available in python 3.10+ and we still support 3.8+, will want to use Union